pyaptamer/aptanet/_pipeline.py

```diff
@@ -58,7 +58,7 @@ class AptaNetPipeline:
 
     def __init__(self, k=None, classifier=None):
         self.k = k
-        self.classifier = classifier or AptaNetClassifier()
+        self.classifier = classifier or AptaNetPredictor()
```

Comment:
the class should be dataclass like - do not overwrite `self.classifier`, instead use `self._classifier` -pre-existing issue, but I am noticing only now